### DISCUSSION TITLE
DWMBG gui applied a border to itself on Win11?

### DISCUSSION THREAD

**Discussion by Sand216** [Maplespe/DWMBlurGlass#335] [General]
I was working on [my project](https://github.com/Maplespe/DWMBlurGlass/discussions/306), referencing the DWMBG gui's design, when I realized something was off. There was miraculously a window border on the gui? This confuses me to no end. I'm assuming this is unintentional, as Maplespe hasn't said anything about window borders. What surprises me is that it just... works! It has rounded corners, the blur effects are applied, and there aren't any visual glitches. Windows 11 removed the window border code in non-high contrast themes, right? How is it possible that a window border was applied? After taking a screenshot, I closed out of the gui and reopened it, and that removed the window border. Can anyone more experienced with DWM and Windows provide some information? Thanks!

![image](https://github.com/Maplespe/DWMBlurGlass/assets/164842290/3d59ceed-174e-4e86-85ff-7bac3f6ed79f)

**Comment by Sand216** [comment]
Bump, can anyone provide some insight? Has this happened to anyone else, or was it just a fluke?

**Comment by watchofficial** [comment]
Same bug is happened on my Windows 10 22H2 device.
The border is weird

**Reply by Sand216** [reply]
The borders are a feature on Windows 10 that can be turned off in the DWMBG settings. I’m saying it is abnormal for borders to appear on Windows 11. All the ways to get window borders on 10 do not work on 11, such as third party themes or the official Aero Lite theme. The only way to get window borders on windows 11 without modifying system files is to use a high contrast theme. This is why I am confused on how a window border was applied.

**Comment by ALTaleX531** [comment]
The fact that no one knows what's causing this, I'm more likely to believe it's a bug of MiaoUI.
